locales/lt-LT/messages.po

```diff
@@ -1575,6 +1575,9 @@ msgstr "Įvertinimo formatas nėra skaitmeninis"
 msgid "The selected file is ready to be sent."
 msgstr "Pasirinktas failas paruoštas išsiųsti."
 
+msgid "Your file has been uploaded."
```

Comment:
What about EN locale? Why I don't see it there?

Reply:
and what about the old string 'The selected file is ready to be sent'? I see that it was translated for all locales